shared/config.ts

```diff
@@ -1,8 +1,20 @@
 import { config } from 'dotenv';
 
+interface WalletConfig {
+  name: string;
+  addressCount: number;
+  seed: {
+    length: 12 | 15 | 18 | 21 | 24;
+    shareCount: number;
+    thresholdCount: number;
+    fileName: (share: number) => string;
+  };
+}
+
 class ConfigClass {
   api = {
     url: process.env.API_URL,
+    // TODO: use wallet address/signature?
     address: process.env.API_ADDRESS,
     signature: process.env.API_SIGNATURE,
```

Comment:
TODO?
My take, I wouldn't move it into wallet object. As this is information regarding the API. Maybe we should improve the names of those parameters

Reply:
I was just thinking that instead of hardcoding address and signature, we could use a cold wallet address to log in to the API. WDYT?

Reply:
ah that what was meant... hm, I don't know. yes why not, doesn't speak anything against that